src/amo/components/CategoryHead/index.js

```diff
@@ -0,0 +1,99 @@
+/* @flow */
+import config from 'config';
+import invariant from 'invariant';
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { getCanonicalURL } from 'amo/utils';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import translate from 'core/i18n/translate';
+import type { AppState } from 'amo/store';
+import type { I18nType } from 'core/types/i18n';
+
+type CategoryType = {|
+  application: string,
+  description: string,
+  id: number,
+  misc: boolean,
+  name: string,
+  slug: string,
+  type:
+    | 'extension'
+    | 'theme'
+    | 'dictionary'
+    | 'search'
+    | 'language'
+    | 'persona',
+  weight: number,
+|};
+
+type Props = {|
+  category: CategoryType | null,
+  type: 'extension' | 'theme' | null,
```

Comment:
will type ever be null?

Reply:
I am not sure, maybe not. Let me change that